community/neo4j/src/test/java/upgrade/StoreUpgraderTest.java

```diff
@@ -359,6 +362,27 @@ public void upgradeShouldNotLeaveLeftoverAndMigrationDirs() throws Exception
     }
 
     @Test
+    public void upgradeShouldGiveProgressMonitorProgressMessages() throws Exception
```

Comment:
We could, but it would be identical, as the counts steps will simply be really fast in that case.